src/Symfony/Component/OptionsResolver/OptionsResolver.php

```diff
@@ -367,6 +383,59 @@ public function getDefinedOptions()
     }
 
     /**
+     * Freeze the option with the given name.
+     *
+     * @param string|string[] $optionNames One or more option names
```

Comment:
yes, thanks to [type casting](https://github.com/symfony/symfony/pull/17161/files#diff-5ec124b2f66baead89d30d56018d8241R400)